src/components/SubscriptionsModal/SubscriptionsModal.tsx

```diff
@@ -349,13 +349,11 @@ export const SubscriptionsModal: React.FC<SubscriptionsModalProps> = ({ isModalO
                   color="primary"
                   size="large"
                   disableElevation
-                  disabled={!selectedClientApp.subscriptions.length || !selectedClientApp.id?.toString().length}
+                  disabled={!!selectedClientApp.subscriptions.length || !selectedClientApp.id?.toString().length}
```

Comment:
Would this work as expected?
```suggestion
                  disabled={!!selectedClientApp.subscriptions.length || !selectedClientApp.id}
```

Reply:
It does seem to work without issues. I'll change it.